app/move/views/_includes/assessment.njk

```diff
@@ -1,4 +1,5 @@
 {% for assessmentCategory in assessment %}
+<section class="app-section" data-app-section="{{ assessmentCategory.key }}">
```

Comment:
It would be good to keep indentation levels for readability.

Reply:
Done (I think)